blackbox/docs/sql/statements/deallocate.rst

```diff
@@ -0,0 +1,40 @@
+.. _ref-deallocate:
+.. highlight:: psql
+
+==============
+``DEALLOCATE``
+==============
+
+Deallocate a prepared statement
+
+.. rubric:: Table of Contents
+
+.. contents::
+   :local:
+
+Synopsis
+========
+
+::
+
+    DEALLOCATE [PREPARE] { name | ALL }
+
+Description
+===========
+
+DEALLOCATE is used to deallocate a previously prepared SQL statement and free
+the reserved resources. It is not meant to be explicitly issued by a user but
+it's used by some clients (e.g. `libpq`_) over Postgres Wire Protocol as an
```

Comment:
@matriv maybe link "Postgres Wire Protocol" to  https://github.com/crate/crate/blob/master/blackbox/docs/interfaces/postgres.rst

Reply:
I'm not sure... I was thinking to link from that file to here somehow... Waiting for @nslater  suggestion too.

Reply:
yeah that looks fine to me